blaze-server/src/test/scala/org/http4s/server/blaze/Http1ServerStageSpec.scala

```diff
@@ -413,6 +413,20 @@ class Http1ServerStageSpec extends Http4sSpec {
         val results = dropDate(ResponseParser.parseBuffer(buff))
         results._1 must_== InternalServerError
       }
+
+      "Handle link header" in {
+        val linkHeader = Link(uri("/feed"))
+
+        val service = HttpService[IO] {
+          case req => IO.pure(Response(body = req.body).replaceAllHeaders(linkHeader))
+        }
+
+        val req = "GET /foo HTTP/1.1\r\n\r\n"
+
+        val buf = Await.result(runRequest(Seq(req), service), 5.seconds)
+        val (_, hdrs, _) = ResponseParser.apply(buf)
+        hdrs.find(_.name == Link.name) must beSome(linkHeader)
+      }
```

Comment:
This header shouldn't really need to be tested in blaze.

If you can add an `Arbitrary[Link]` (I'll help if you can't), then you can follow a model more like the [other header specs](https://github.com/http4s/http4s/blob/master/tests/src/test/scala/org/http4s/headers/ContentLengthSpec.scala).  It'll be a simple test that covers more.